spec/bourbon/css3/font_face_spec.rb

```diff
@@ -15,7 +15,8 @@
                 "url(\"/fonts.woff2\") format(\"woff2\"), " +
                 "url(\"/fonts.woff\") format(\"woff\"), " +
                 "url(\"/fonts.ttf\") format(\"truetype\"), " +
-                "url(\"/fonts.svg#Helvetica\") format(\"svg\");; " +
+                "url(\"/fonts.svg#Helvetica\") format(\"svg\"); " +
```

Comment:
Use 2 (not 10) spaces for indenting an expression in an assignment spanning multiple lines.